packages/url_launcher/url_launcher/README.md

```diff
@@ -96,16 +96,18 @@ takes a string argument containing a URL. This URL
 can be formatted using a number of different URL schemes. The supported
 URL schemes depend on the underlying platform and installed apps.
 
-Common schemes supported by both iOS and Android:
+Common schemes supported:
```

Comment:
Let's changed this to "Commonly used schemes include:"

"Supported" sends the wrong message here (which is addressed in the sentence above, but we shouldn't muddy the waters here).